io/net/ethtool_test.py

```diff
@@ -212,6 +212,14 @@ def test_ethtool(self):
                                       ignore_status=True)
                     if ret.exit_status != 0:
                         self.fail("%s %s" % (self.args, ret.stderr_text))
+            elif self.args == "-e":
+                cmd = "ethtool %s %s >> /tmp/eeprom.log 2>&1" % (self.args, self.interface)
+                ret = process.run(cmd, shell=True, verbose=True, ignore_status=True)
+                if ret.exit_status != 0:
+                    if "Operation not supported" in ret.stderr_text:
+                        self.log.warn("%s failed" % self.args)
```

Comment:
@PavamanSubramaniyam I think we need to cancel this if not supported.
please re-check on this